plugins/kubernetes/src/components/CustomResources/ArgoRollouts/Rollout.tsx

```diff
@@ -0,0 +1,271 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useContext } from 'react';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Divider,
+  Grid,
+  Typography,
+} from '@material-ui/core';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { V1Pod, V1HorizontalPodAutoscaler } from '@kubernetes/client-node';
+import { StatusError, StatusOK } from '@backstage/core';
+import { PodsTable } from '../../Pods';
+import { HorizontalPodAutoscalerDrawer } from '../../HorizontalPodAutoscalers';
+import { RolloutDrawer } from './RolloutDrawer';
+import PauseIcon from '@material-ui/icons/Pause';
+import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
+import formatDistance from 'date-fns/formatDistance';
```

Comment:
Not sure but I think this may apply here?

https://backstage.io/docs/architecture-decisions/adrs-adr010

Reply:
Thanks, I'll take a look!

Reply:
Sound be fixed